src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -0,0 +1,203 @@
+    />
+
+    </Collapser>
+    <Collapser
+        id="bulk-change-status"
+        title="Remove Ignored status across multiple instances"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 
+1. From the settings cog, select the **Ignored** checkbox. This adds an ignore column to your vulnerability table.
+1. Select a vulnerability with an ignored count greater than 0. 
+
+    <img
+        title="Vulnerabilities tab with ignored column added"
+        alt="An image showing the all entities vulnerabilities tab with an ignore column added"
+        src={vmAffectedBulkVulnList}
+    />
+
+You can also bulk manage your Ignored vulnerabilities with the filter bar. This method is better if you X:
+
+1. Update the filter from `Status = Affected` to `Status = Ignored`. 
+
+    <img
+        title="Vulnerabilities details"
+        alt="An image showing the vulnerability details page with the status dropdown open"
+        src={vmAffectedBulkDetails}
+    />
+
+1. Referring to the **Area of Impact** section, select the entities you want to mark Affected.
+
+    <img
+        title="Vulnerabilities details with instances selected"
+        alt="An image showing the vulnerability details page with affected instances selected"
+        src={vmAffectedBulkSelection}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Affected.
+
+    <img
+        title="Vulnerabilities details with form open"
+        alt="An image showing the vulnerability details page with form to change instances to affected"
+        src={vmAffectedBulkForm}
+    />
+
+    </Collapser>
+</CollapserGroup>
+
+## Surface Ignored vulnerabilities [#surface-vulnerabilities]
+
+Before you assign **Ignored** to a vulnerability or entity, it's important to understand that the change affects how other code owners manage that particular instance. For example, marking a vulnerability or entity as **Ignored** prevents other users with account access from seeing that vulnerability. 
+
+* You can surface **Ignored** vulnerabilities with the filter bar. From the filter bar, you can add `Status = Ignored` to view Ignored vulnerabilities. 
+* As best practice, we recommend reviewing your vulnerabilities at minimum once every 90 days to ensure you aren't introducing added risk into your system.
+
+## Why ignore a vulnerability? [#why-ignore]
+
+When you choose to ignore a vulnerability, you'll be prompted to explain the rationale behind the decision. For example, maybe your system uses some package that contains a mixture of deprecated and supported code. Your system may depend on the supported code but not the deprecated code. You can choose to ignore the vulnerability notice as it's irrelevant to your system. This is good context to provide in the form, and we recommend you provide as much documentation to corroborate it as possible. 
```

Comment:
I don't think this phrased correctly. 
Deprecated code and supported code is different than 'not using the vulnerable function'. 

I think this section should be reworded again to target the 'vulnerable method is not used'.
Happy to hop on a call to chat about this again.

Reply:
sent you example/updated copy via slack!